pom.xml

```diff
@@ -111,4 +111,15 @@
       <url>https://repo.jenkins-ci.org/public/</url>
     </pluginRepository>
   </pluginRepositories>
+
+  <build>
+    <plugins>
+      <plugin>
+        <artifactId>maven-surefire-plugin</artifactId>
+        <configuration>
+          <argLine>-Dfile.encoding=${project.build.sourceEncoding}</argLine>
```

Comment:
If you had to do this, I suspect the test is at fault.

Reply:
No, this really is a Surefire bug: [Surefire never outputs UTF-8 under Windows](https://issues.apache.org/jira/browse/SUREFIRE-1220)
I can reproduce it all with the attached sample project up to version `3.0.0-M3` which ansicolor-plugin currently uses. 

I will see if this can also be dealt with simply by updating Surefire.

Reply:
That is about test _output_. I am talking about the fact that the test apparently _passes or fails_ depending on system encoding.

Reply:
Freestyle projects are unduly sensitive to master encoding. Pipeline projects as of [JEP-206](https://jenkins.io/jep/206) are not.

Reply:
Anyway, I suppose there is not an easy way to make the test behave reproducibly without either using Pipeline (which this wrapper is not compatible with), or running all tests in UTF-8 encoding (as you did).